src/chart/generateCategoricalChart.js

```diff
@@ -44,6 +44,7 @@ const generateCategoricalChart = (ChartComponent, GraphicalChild) => {
     static propTypes = {
       ...ChartComponent.propTypes,
       syncId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+      compact: PropTypes.bool,
```

Comment:
@evisong About this name of this props, do you have any idea when user want to use this props to generate svg elements (something describe in [this issue](https://github.com/recharts/recharts/issues/205)) ?